src/SwedbankPay.Sdk/Payments/Swish/Payment.cs

```diff
@@ -18,13 +20,12 @@ private Payment(PaymentResponse paymentResponse, SwedbankPayHttpClient client)
                 {
                     case PaymentResourceOperations.UpdatePaymentAbort:
                         operations.Abort = async () =>
-                            await client.SendHttpRequestAndProcessHttpResponse<PaymentResponse>(
-                                httpOperation.Request.AttachPayload(new PaymentAbortRequest()));
+                            await client.PatchAsJsonAsync<PaymentResponse>(httpOperation.Href, new PaymentAbortRequest());
```

Comment:
This does not follow the operations design. Here we assume it's a Patch again instead of using the Method from the Operations response.

Reply:
I see.
I'll find a way to introduce that back in the code.